### ISSUE TITLE
U2F support for steam's 2 factor authentication

### ISSUE THREAD

**Issue by pilotforhire** [ValveSoftware/steam-for-linux#4521]
Dear Valve,

you have been supporting 2 factor authentification via mobile phone so far. I think this is a security problem because smartphones are stolen very often. If someone steals my phone, he will have access to my steam account.
To prevent that scenario I request U2F usb dongle support in steam as an alternative. That will be more secure because you can store your usb stick in a safe place. A cell phone is something that people will always have with them so it is vulnerable for theft.

**Comment by ekaris** [comment]
It is up to the manufacturer to support it, please close this issue, YubiKey already added support for steam for example:
https://github.com/Yubico/yubioath-desktop/issues/72

**Comment by pilotforhire** [comment]
No, it is not. I asked for U2F, not a custom Yubikey OATH-Hack.

**Comment by Brianetta** [comment]
My wife and I are proud owners of U2F tokens. Mine protects my Github account; it would be nice if it could also protect my Steam account.

**Comment by sarndt** [comment]
voicing my support for this idea! U2F should be made availabe, I for one can't use my smartphone for authentication but have a u2f usb dongle lying around that I would gladly use

**Comment by psychotrome** [comment]
Same here.

Just got my yubikey and pissed that I can't use it with Steam because of Valves backwards implementation.

**Comment by hannesrohde** [comment]
"Me too" - have a Yubikey Neo and would like to see Valve endorse and properly support it. The suggested solution of copying the secret from the Steam android app and pasting it somewhere is a hack and not very convenient

**Comment by jabbera** [comment]
Bump

**Comment by Daramulun** [comment]
I have a Yubikey4Nano, Yubikey4 and YubikeyNeo and definitely want steam to finally support U2F, don't understand why they not added it, since they already use chromium for the steam client it should be easy to add.

**Comment by cinhtau** [comment]
👍

**Comment by chuckie512** [comment]
+1

**Comment by eth0up** [comment]
+1

**Comment by Arnros** [comment]
+1

**Comment by freddyyeddy** [comment]
+1

**Comment by joydashy** [comment]
Two years, and still no pulse from Valve? This is what you get in a company where people can work on whatever they please and have no direction.

**Comment by Galtrox** [comment]
I wish they would add this feature would indeed be nice :(

**Comment by 13lackWidow** [comment]
+1, just acquired a yubikey for myself and would love to see this implemented.

**Comment by victoredwardocallaghan** [comment]
+1

**Comment by mindforger** [comment]
is this still not a thing?

**Comment by odlevakp** [comment]
+1

**Comment by daemith** [comment]
+1

**Comment by OlliC** [comment]
+1

**Comment by nesttle** [comment]
Implement this ASAP please, is not difficult,

**Comment by KillaBoi** [comment]
this issue has been open for 4 years now, i think its time to actually do this.

**Comment by chiliseek** [comment]
+1 ... could you please implement this?

**Comment by satolas** [comment]
Indeed would be good to have :-)

**Comment by bl3chd0se** [comment]
+1 would be really nice to have this

**Comment by koutsie** [comment]
+1 would be good to have.

**Comment by Spontifex** [comment]
+1 I completely agree that adding support for U2F authentication to Steam's 2FA system would provide an additional layer of security for users. Physical security keys, such as USB dongles, can be stored in a safe place and are less vulnerable to theft than mobile phones, which are often carried with us at all times. I also think it would be a valuable addition to Steam's authentication options, and I hope that the Steam development team will consider implementing U2F support in the future.
